Last-Update: 2012-08-24
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: backported patch (r31566)
 Rearrange date and time formats.
 Order them: date, time, date and time.
 Arrange international formats before American ones.

--- a/src/wp/ap/xp/ap_Dialog_Insert_DateTime.cpp
+++ b/src/wp/ap/xp/ap_Dialog_Insert_DateTime.cpp
@@ -74,23 +74,23 @@
 };
 #else
 const char *InsertDateTimeFmts[] = {
-    "%m/%d/%y",
-	"%A, %B %d, %Y",
-	"%B %d, %Y",
-	"%m/%d/%Y",
 	"%Y-%m-%d",
-	"%d-%b-%y",
-    "%m.%d.%y",
-	"%b. %d, %y",
-	"%d %B, %Y",
+	"%m/%d/%Y",
+	"%m/%d/%y",
+	"%m.%d.%y",
 	"%B, %Y",
 	"%b-%y",
-	"%m/%d/%y %I:%M %p",
-    "%m/%d/%y %I:%M:%S %p",
-	"%I:%M %p",
-	"%I:%M:%S %p",
+	"%B %d, %Y",
+	"%b. %d, %y",
+	"%d %B, %Y",
+	"%d-%b-%y",
+	"%A, %B %d, %Y",
 	"%H:%M",
 	"%H:%M:%S",
+	"%I:%M %p",
+	"%I:%M:%S %p",
+	"%m/%d/%y %I:%M %p",
+	"%m/%d/%y %I:%M:%S %p",
 	NULL
 };
 #endif
